src/index.css

```diff
@@ -1,13 +1,120 @@
 :root {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #191b1b;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+body {
+  margin: 0;
+}
+.album {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 15px;
+  box-sizing: border-box;
+}
+.album-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
```

Comment:
This doesn't look too good when there's a "bastard" at the last row:

![Skärmavbild 2024-03-13 kl  11 06 33](https://github.com/Technigo/project-music-releases-vite/assets/51529201/35e24226-0a16-4728-9307-cc614391cfe3)